staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -580,6 +580,40 @@ type PatchOptions struct {
 	FieldManager string `json:"fieldManager,omitempty" protobuf:"bytes,3,name=fieldManager"`
 }
 
+// ApplyOptions may be provided when applying an API object.
+// ApplyOptions is meant to be a subset of PatchOptions that does not include FieldManager.
```

Comment:
nit: Is this comment still accurate?

Edit: Actually just want to confirm, do we still want to have a separate applyoptions or just directly use patchoptions?

Reply:
I still want a separate one for future proofing. E.g. if we add something to patch options that doesn't make sense to apply options, or if we want to customize the godoc to better explain apply, it's nice to have a separate type. I realize that, at the moment, it doesn't seem particularly useful. But it does look nice in code to have the options match the operation, so even if we don't ever really take advantage of it, I'm still OK with it.

Reply:
Fixing the doc error.